Firmware/config.h

```diff
@@ -59,7 +59,9 @@
 //#define LANG_MODE              0 // primary language only
 #define LANG_MODE              1 // sec. language support
 
-#define LANG_SIZE_RESERVED     0x3000 // reserved space for secondary language (12288 bytes). Maximum 32768 bytes
+#define LANG_SIZE_RESERVED     0x3900 // reserved space for secondary language (14592 bytes).
```

Comment:
@3d-gussner are we ok with this size? The MMU error messages are not here yet, I don't see a reson to enlarging this number here.